src/components/SearchForm.jsx

```diff
@@ -63,16 +81,26 @@ export const FormikSearch = withFormik({
     searchTerms,
     location,
     radius,
-    timeFrame
+    timeFrame,
+    savedSearch,
+    plan,
   }) {
     {/* return an object which sets initial values of the form */}
     return {
-      searchTerms: '',
-      location: '97015',
-      radius: '',
-      timeFrame: ''
+      searchTerms: searchTerms || '',
+      location: location || '97015',
+      radius: radius || '',
+      timeFrame: timeFrame || '',
+      savedSearch: savedSearch || true,
+      plan: plan || 'free',
     }
   },
+  validationSchema: Yup.object().shape({
+    searchTerms: Yup.string().required(),
+    location: Yup.string().min(5).max(5),
+    radius: Yup.string(),
```

Comment:
I'm still reading about Yup, but from the documentation it seems like we can do:
```
radius: Yup.number()
```
Or depending on what we want even:
```
radius: Yup
  .number()
  .positive()
  .max(50, 'You have exceeded the maximum distance)
  .notRequired(),
```